src/rust/bitbox02/src/screen_saver.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Shift Cryptosecurity AG
+// Copyright 2020 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// calls the C function in screen_saver.c to enable the screen saver
```

Comment:
fyi in Rust, docstrings have three slahes `///`. Not a big deal though as we don't generate docs anyway :P